tests/unittests/testFuselageStandardProfileSuperellipse.cpp

```diff
@@ -0,0 +1,174 @@
+        tiglRet1 = tiglOpenCPACSConfiguration(tixiHandle1, "", &tiglHandle1);
+        ASSERT_TRUE(tiglRet1 == TIGL_SUCCESS);
+
+        // Test case on standardProfile, invalid elements
+
+        ReturnCode tixiRet2;
+        TiglReturnCode tiglRet2;
+
+        tiglHandle2 = -1;
+        tixiHandle2 = -1;
+
+        tixiRet2 = tixiOpenDocument(filename1, &tixiHandle2);
+        ASSERT_TRUE (tixiRet2 == SUCCESS);
+        tiglRet2 = tiglOpenCPACSConfiguration(tixiHandle2, "", &tiglHandle2);
+        ASSERT_TRUE(tiglRet2 == TIGL_SUCCESS);
+
+
+    }
+
+    static void TearDownTestCase()
+    {
+        ASSERT_TRUE(tiglCloseCPACSConfiguration(tiglHandle) == TIGL_SUCCESS);
+        ASSERT_TRUE(tixiCloseDocument(tixiHandle) == SUCCESS);
+        tiglHandle = -1;
+        tixiHandle = -1;
+
+        ASSERT_TRUE(tiglCloseCPACSConfiguration(tiglHandle1) == TIGL_SUCCESS);
+        ASSERT_TRUE(tixiCloseDocument(tixiHandle1) == SUCCESS);
+        tiglHandle1 = -1;
+        tixiHandle1 = -1;
+
+        ASSERT_TRUE(tiglCloseCPACSConfiguration(tiglHandle2) == TIGL_SUCCESS);
+        ASSERT_TRUE(tixiCloseDocument(tixiHandle2) == SUCCESS);
+        tiglHandle2 = -1;
+        tixiHandle2 = -1;
+
+    }
+
+    void SetUp() override {}
+    void TearDown() override {}
+
+
+    static TixiDocumentHandle           tixiHandle;
+    static TiglCPACSConfigurationHandle tiglHandle;
+
+    static TixiDocumentHandle           tixiHandle1;
+    static TiglCPACSConfigurationHandle tiglHandle1;
+
+
+    static TixiDocumentHandle           tixiHandle2;
+    static TiglCPACSConfigurationHandle tiglHandle2;
+};
+
+TixiDocumentHandle FuselageStandardProfileSuperEllipse::tixiHandle = 0;
+TiglCPACSConfigurationHandle FuselageStandardProfileSuperEllipse::tiglHandle = 0;
+TixiDocumentHandle FuselageStandardProfileSuperEllipse::tixiHandle1 = 0;
+TiglCPACSConfigurationHandle FuselageStandardProfileSuperEllipse::tiglHandle1 = 0;
+TixiDocumentHandle FuselageStandardProfileSuperEllipse::tixiHandle2 = 0;
+TiglCPACSConfigurationHandle FuselageStandardProfileSuperEllipse::tiglHandle2 = 0;
+
+TEST_F(FuselageStandardProfileSuperEllipse, BuildWireSuperEllipse)
+{
+    //test valid values
+    auto wire = BuildWireSuperEllipse(0.25,5.,0.5,3.,2);
+    ASSERT_TRUE(wire.Closed());
+    auto trafo = gp_Trsf();
+    auto vec = gp_Vec(-1.,0.,0.);
+    trafo.SetTranslation(vec);
+    auto wire2 = BRepBuilderAPI_Transform(wire, trafo).Shape();
+    ASSERT_TRUE(wire2.Closed());
+    auto loft = CTiglMakeLoft();
+    loft.addProfiles(wire);
+    loft.addProfiles(wire2);
+    ASSERT_TRUE(BRepCheck_Analyzer(loft.Shape()).IsValid());
+
+    //check invalid values
+    ASSERT_THROW(BuildWireSuperEllipse(0.25,5.,0.5,3.,0.), tigl::CTiglError);
+    ASSERT_THROW(BuildWireSuperEllipse(0.,5.,0.5,3.,2.), tigl::CTiglError);
+    ASSERT_THROW(BuildWireSuperEllipse(1.,5.,0.5,3.,2.), tigl::CTiglError);
+    ASSERT_THROW(BuildWireSuperEllipse(0.25,0.,0.5,3.,2.), tigl::CTiglError);
```

Comment:
Would you mind checking negative exponents as well as lowerHeightFraction with invalid values?